src/Storage/TSStorageManager.m

```diff
@@ -197,8 +197,10 @@ - (void)setupDatabase
 {
     // Register extensions which are essential for rendering threads synchronously
     [TSDatabaseView registerThreadDatabaseView];
-    [TSDatabaseView registerBuddyConversationDatabaseView];
+    [TSDatabaseView registerThreadInteractionsDatabaseView];
```

Comment:
Sorry, sloppy comment, let me try again. =)

We're making a couple new DB views. Each of these will iterate over every object in the data store, which can take some time.

During QA, we should verify the upgrade process is reasonably smooth for slow devices that have huge message collections.